contracts/application/AppDirectory.sol

```diff
@@ -7,8 +7,7 @@ import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 /**
  * @title AppDirectory
  * @dev Implementation directory with a standard library as a fallback provider.
- * If the implementation is not found in the directory, it will search in the
- * standard library.
+ * @dev If the implementation is not found in the directory, it will search in the standard library.
```

Comment:
Regarding `@dev` there is a good explanation about it here: https://github.com/OpenZeppelin/openzeppelin-solidity/pull/995#issuecomment-396431216

Reply:
I don't understand the comments about `max-len`. These are all below 120. Can you clarify what change you're suggesting, if any?

Reply:
Sorry I was unclear, I was simply asking whether the `max-len` is indeed 120, because these comments were recently edited (and split in several lines) by @elopio, who is on top of the style guides.. 

If it is indeed 120, it's good as it is (in the PR).

Reply:
👍

Reply:
Oh! But I did suggest removing the second @dev.